nsis/SharedBase.nsh

```diff
@@ -38,7 +38,8 @@ SetCompressorDictSize 24
 ; The name of the installer
 Name "${APP_NAME}"
 
-OutFile "output\pcsx2-${APP_VERSION}-${OUTFILE_POSTFIX}.exe"
+; Output the installer to the nsis folder (in git repo)
+OutFile "pcsx2-${APP_VERSION}-${OUTFILE_POSTFIX}.exe"
```

Comment:
You'll need to update the gitignore to reflect this change.

Why not have NSIS automatically create the output directory if it doesn't exist?